### ISSUE TITLE
Error when processing intermediate results

### ISSUE THREAD

**Issue by themonk911** [GoogleCloudPlatform/covid-19-open-data#46]
```
"Traceback (most recent call last):
  File "/env/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/env/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/env/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/vmagent/app/appengine.py", line 221, in update_table
    output_folder / "intermediate", intermediate_results
  File "/home/vmagent/app/lib/pipeline.py", line 299, in _save_intermediate_results
    export_csv(result, intermediate_folder / file_name, schema=self.schema)
  File "/home/vmagent/app/lib/io.py", line 260, in export_csv
    data_fmt[col] = data[col].apply(lambda val: "" if pandas.isna(val) else fmt(val))
  File "/env/lib/python3.7/site-packages/pandas/core/series.py", line 3848, in apply
    mapped = lib.map_infer(values, f, convert=convert_dtype)
  File "pandas/_libs/lib.pyx", line 2329, in pandas._libs.lib.map_infer
  File "/home/vmagent/app/lib/io.py", line 260, in <lambda>
```

**Comment by owahltinez** [comment]
Is there any way to get the GET request which triggered the error? I can see from the logs that it's coming from a call to `update_table`, and it looks like it happened 14 times so it cannot be more than a single data source which is causing this.

**Comment by owahltinez** [comment]
Fixed via #48, hopefully. Let's keep it open until we verify that the errors go away.
